test/test-utils/helpers/moveHelper.ts

```diff
@@ -218,7 +218,7 @@ export class MoveHelper extends GameManagerHelper {
       "SelectTargetPhase",
       UiMode.TARGET_SELECT,
       () => {
-        const handler = this.game.scene.ui.getHandler() as TargetSelectUiHandler;
+        const handler = this.game.scene.ui.getCurrentHandler() as TargetSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = this.game.scene.ui.getCurrentHandler<TargetSelectUiHandler>();
```